fastchat/serve/monitor/monitor.py

```diff
@@ -442,8 +443,11 @@ def build_arena_tab(
     for k in key_to_category_name.keys():
         if k not in elo_results:
             continue
-        arena_dfs[key_to_category_name[k]] = elo_results[k]["leaderboard_table_df"]
-        category_elo_results[key_to_category_name[k]] = elo_results[k]
+        category_name = key_to_category_name[k.replace("_style_control", "")]
+        if "_style_control" in k:
+            category_name = f"{category_name} w/ Style Control"
```

Comment:
Sorry, but could you explain what's going on here?

Reply:
ah yeah. Based on the previous code logic the stylecontrol leaderboards have the _style_control in their key. Instead of having duplicates for every category name in monitor_md like we have now we add it in here.